Sources/Sham/MockService.swift

```diff
@@ -8,7 +8,7 @@ public class MockService {
     // MARK: - Shared Instance
     
     /// The shared instance of the service.
-    public static let shared = MockService()
+    public static var shared = MockService()
```

Comment:
Could we leave this as a constant and just inject the stubbed collection through a method (or mapping over the collection and injecting each individual stub)?

Just thinking how making this mutable could lead to API misuse (e.x. making changes to `MockService.shared.defaultBundle` prior to calling `setupWithLaunchEnvironment`, which would override it back to its defaults.)